tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.base.json",
```

Comment:
What is this tsconfig being used for?

Reply:
This is used for the scripts in the `scripts` folder